### PR TITLE
Add unit tests for com.linkedin.tony.util.Utils

### PR DESCRIPTION
**Pull request by EricHetti**
I've analysed your codebase and noticed that `com.linkedin.tony.util.Utils` is not fully tested.
I've written some tests for the methods in this class with the help of [Diffblue Cover](https://www.diffblue.com/opensource).

Hopefully, these tests will help you detect any regressions caused by future code changes. If you would find it useful to have additional tests written for this repository, I would be more than happy to look at other classes that you consider important in a subsequent PR.

I also refactored a few existing `assertEquals` statements to ensure that the `actual` value comes first

### CODE REVIEW

**Review by JohnLBergqvist — CHANGES_REQUESTED** [review]
I would prefer you didn't move the existing stuff around (at least, not in the same commit) as it makes what you've added hard to determine.

**Comment by Braavos6** [comment]
> I would prefer you didn't move the existing stuff around (at least, not in the same commit) as it makes what you've added hard to determine.

Ok I moved them back :)

File: tony-core/src/test/java/com/linkedin/tony/util/TestUtils.java [hunk 1]
```diff
@@ -7,119 +7,263 @@
...
-  }
-
-  @Test
-  public void testBuildRMUrl() {
-    Configuration yarnConf = mock(Configuration.class);
-    when(yarnConf.get(YarnConfiguration.RM_WEBAPP_ADDRESS)).thenReturn("testrmaddress");
-    String expected = "http://testrmaddress/cluster/app/1";
-    assertEquals(Utils.buildRMUrl(yarnConf, "1"), expected);
-  }
+
```
Comment (Braavos6) [inline tony-core/src/test/java/com/linkedin/tony/util/TestUtils.java:33]: Not sure why this happens....

Comment (Braavos6) [reply tony-core/src/test/java/com/linkedin/tony/util/TestUtils.java:33]: I will clone the project and try again

Comment (Braavos6) [reply tony-core/src/test/java/com/linkedin/tony/util/TestUtils.java:33]: Done.... project uses 2 spaces -_-

**Review by JohnLBergqvist — APPROVED** [review]

**Comment by JohnLBergqvist** [comment]
@Braavos96 Split out the changes to their tests into a seperate commit.

**Review by paulkennethkent — APPROVED** [review]

**Review by JohnLBergqvist — APPROVED** [review]

**Pull request closed without merging**